ci/scripts/rust_clippy.sh

```diff
@@ -18,4 +18,4 @@
 # under the License.
 
 set -ex
-cargo clippy --all-targets --workspace -- -D warnings
+cargo clippy --all-targets --workspace --features avro,jit,pyarrow,scheduler -- -D warnings
```

Comment:
I tried running with 

```suggestion
cargo clippy --all-targets --workspace --all-features -- -D warnings
```


```
error[E0554]: `#![feature]` may not be used on the stable release channel
   --> /Users/alamb/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd_2-0.3.8/src/lib.rs:214:1
    |
214 | / #![feature(
215 | |     adt_const_params,
216 | |     repr_simd,
217 | |     rustc_attrs,
...   |
224 | |     custom_inner_attributes,
225 | | )]
    | |__^
```

Reply:
Sadly there doesn't appear to be an except feature flag